### ISSUE TITLE
[suggestion] Allow for category creation which is consensus driven

### ISSUE THREAD

**Issue by amdza12** [graphprotocol/mission-control-curator#321]
Not sure if this has been mentioned earlier but currently the canvas is pretty wide for tokenization of existing world assets and web 2.0 The curator community should have access to be able to paint with a more open palette. Wonder if its possible to allow for creation of new Categories and Subcategories. Some of the existing categories like Blockchain seem very broad and do not reflect the diversity of projects listed under them. This could of-course be gamed, so to make it sybil-resistant some sort of voting scheme, similar to project creation/voting could be enforced.

**Comment by yanivtal** [comment]
Yes categories should for sure be something that curators can curate. Would like to see a good implementation of this.

**Comment by yanivtal** [comment]
Duplicate of https://github.com/graphprotocol/mission-control-curator/issues/284
